astropy/io/ascii/ui.py

```diff
@@ -144,13 +144,13 @@ def get_reader(Reader=None, Inputter=None, Outputter=None, **kwargs):
     names : list
         List of names corresponding to each data column
     include_names : list
-        List of names to include in output (default= ``None`` selects all names)
```

Comment:
If you use single tick instead of double, Sphinx will link to the variable API. Or is that also not readable?